nbgrader/server_extensions/formgrader/apihandlers.py

```diff
@@ -272,13 +273,54 @@ def post(self, assignment_id, student_id):
         self.write(json.dumps(self.api.autograde(assignment_id, student_id)))
 
 
+class GenerateAllFeedbackHandler(BaseApiHandler):
```

Comment:
Can you add the `@check_notebook_dir` decorator to all these new routes? (Sorry, it's from a PR I just merged recently)

Reply:
Done